pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: purchases_flutter
-description: A Flutter plugin that makes it simple to build in-app purchases and subscriptions – receipt validation, status tracking, and more. Supports iOS and Android.
+description: A Flutter plugin that makes it simple to build in-app purchases (IAP) and subscriptions. Supports iOS, Android, and Stripe.
```

Comment:
Would this lead to developers to think that we support the Web platform? We've had several requests for it https://github.com/RevenueCat/purchases-flutter/issues/162 . @RevenueCat/sdk what do you think?

Reply:
It might also lead to some confusion since on our pub.dev, Web is marked as not supported, right?

Reply:
Why is marked as not supported?